soc/arm/st_stm32/stm32l0/Kconfig.defconfig.stm32l051xx

```diff
@@ -0,0 +1,15 @@
+# ST Microelectronics STM32L051XX MCU
+
+# Copyright (c) 2018 Anthony Kreft <anthony.kreft@gmail.com>
+# Copyright (c) 2021 Even Falch-Larsen <even.falch.larsen@nomono.co>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32L051XX
+
+config SOC
+	default "stm32l051xx"
+
+config NUM_IRQS
+	default 32
```

Comment:
```suggestion
	default 30
```

Reply:
Yes, the stm32L0x1 has  **30** interrupt entries,
So that  `soc/arm/st_stm32/stm32l0/Kconfig.defconfig.stm32l011xx` 
 and `soc/arm/st_stm32/stm32l0/Kconfig.defconfig.stm32l031xx` are probably wrong.